dev_guide/builds.adoc

```diff
@@ -327,6 +327,32 @@ The following options are specific to the
 link:../architecture/core_concepts/builds_and_image_streams.html#docker-build[Docker
 build strategy].
 
+[[dockerfile-path]]
+
+=== Dockerfile Path
+
+Docker builds normally use a Dockerfile named *_Dockerfile_*, located at the root
+of the context (identified by the `*BuildConfig.spec.source.contextDir*` field).
+The `*dockerfilePath*` field allows the build to use a different path to locate your Dockerfile.
+The path must be relative to the `*BuildConfig.spec.source.contextDir*` field.
+It could be just a different name for your Dockerfile (*_MyDockerfile_* for example),
+or a path to a Dockerfile in a subdirectory (*_dockerfiles/app1_* for example):
+
+====
+
+[source,json]
+----
+{
+  "strategy": {
+    "type": "Docker",
+    "dockerStrategy": {
+      "dockerfilePath": "dockerfiles/app1"
+    }
+  }
+}
+----
+====
```

Comment:
Any chance this example could be in yaml and not json? We're trying to lean towards that in the docs.